src/libs/actions/BankAccounts.js

```diff
@@ -751,7 +753,8 @@ function setupWithdrawalAccount(data) {
                 }
             } else {
                 if (response.jsonCode === 666 || response.jsonCode === 404) {
-                    error = response.message;
+                    error = response.htmlMessage || response.message;
```

Comment:
Can you add a comment here explaining why we're setting htmlMessage first? Not everybody has access to the web codebase so it'd be helpful to let people know these specific cases have an HTML message and the 402/others don't.

Reply:
Something like:
```
// Sometimes, the response will have an error message in html format. We give priority to the html error because it has richer content.
```

About the 402/others below, I guess they could have `htmlMessage`, but we are handling those in a completely different way. We don't show their messages directly. This is inconsistent right now and out of the scope.

Reply:
I'd say something like

```
// Since these specific responses can have an error message in html format with richer content, give priority to the html error.
```

Agreed it's fine to leave the 402 alone, I just meant to add a comment like the one you proposed to highlight the difference. Good stuff 👍

Reply:
I used your comment, thanks 😬